### DISCUSSION TITLE
[git plugin] Add alias/function for deleting remote branches

### DISCUSSION THREAD

**Discussion by cesarcoatl** [ohmyzsh/ohmyzsh#9684] [Ideas]
Hello there!

Clearly I am not well versed for I have attempted this without success, so here it goes.

I have created an alias on my `.gitconfig`, but I don't get branch name completion as I do when I type the full command `git push origin --delete` or `git push origin :`.

Would it be possible to add an alias or function that calls `git push origin --delete <remote-branch>` to the `git` plugin?

Thanks!

**Comment by mcornella** [comment]
git aliases inside `.gitconfig` don't get automatic completion because zsh can't hook into the actual command. You can add a zsh alias to `git push --delete origin` like so:
```zsh
alias <alias_name>='git push --delete origin'
```
